### ISSUE TITLE
Don't test Uint8ClampedArray in %TypedArray% tests

### ISSUE THREAD

**Issue by rakudrama** [compat-table/compat-table#1217]
IE11 shows 'No' for many %TypedArray% features but the failure is due to missing Uint8ClampedArray, and not the missing feature on other typed arrays.

**Comment by ljharb** [comment]
Then IE 11 doesn't fully support TypedArrays.

I'd support splitting the tests up into subtests when applicable, but < 100% is the same as 0% when asking "do you support this feature".

**Comment by rakudrama** [comment]
I understand what you are saying, but I'm not finding the table helpful for the following situation.
Can I replace the following code that copies part of the array and works on IE 11

```js
new Uint8Array(myUint8Array.subarray(1,5))
```

with the following:

```js
 myUnit8Array.slice(1,5)
```

The table does not help me make that determination.

Is there something to be done to give a better reflection of the partial state of support?

**Comment by ljharb** [comment]
Have you tried clicking the triangle to expand the "typed arrays" section?

**Comment by rakudrama** [comment]
Oh yes. The entry for %TypedArray%.prototype.subarray x IE11 says "No".
It says "No" because the (C) snippet crashes on the subexpression Uint8ClampedArray.prototype.subarray.
That doesn't _quite_ capture that I can use subarray on other typed arrays.

**Comment by ljharb** [comment]
Fair enough. We'd be happy to accept a PR that splits up the `subarray` tests so that it's more clear what works and what doesn't on IE 11!
